framework/helpers/BaseFormatConverter.php

```diff
@@ -240,63 +241,63 @@ public static function convertDatePhpToIcu($pattern)
             "'" => "''''",  // single `'` should be encoded as `''`, which internally should be encoded as `''''`
             // Day
             '\d' => "'d'",
-            'd' => 'dd',    // Day of the month, 2 digits with leading zeros 	01 to 31
+            'd' => 'dd',    // Day of the month, 2 digits with leading zeros    01 to 31
```

Comment:
IMO these tabs should be removed or converted to something other than spaces - I don't see the point in converting them to spaces.

Reply:
They were automatically fixed. Should we use `[` and `]` maybe?

Reply:
`—` should be fine.

Reply:
Fixed in ad37ec45caa1b82705c2064c200348dabf535f7c